### PR TITLE
Adds two new snacks! Toritose and Borer Yummies

### PR DESCRIPTION
**Pull request by Super3222**
Inspired by Shadow's shitposting.

#### Intent of your Pull Request

TWO NEW SNACKS!
Toritose - Doritos variant of this universe.
Borer yummies - gooey snacks that explode in your mouth.

I'm also going to need a generous spriters help with spriting 3 sprites. A non-finished version of the toritose and yummies, and than the trash version for the toritose

The borer yummies won't need a trash version. I'm also imaging them to be something like candy worms, but maybe more borer shaped?
For the toritose, I'm imagining a bag like the chips, but maybe a different color? It could mimic actual Doritos or whatever you want to come up with.

- [x] Toritose Sprite
- [x] Toritose Trash Sprite
- [x] Borer Yummie Sprite

##### Changelog

:cl:
rscadd: Shout out to AndySok for creating the sprites for the specific snacks mentioned below!
rscadd: Nanotrasen has implemented two new treats into their snack vendors!
rscadd: Toritose - a crunchy, triangle shaped, nacho flavored refined chip.
rscadd: Borer Yummies - gooey snacks that'll open up a tank transfer valve full of flavour in your mouth.
/:cl:

My head hurts from using toritose instead of tortoise.

### CODE REVIEW

**Comment by Super3222** [comment]
In the last commit:

- AndySok was generous enough to sprite all the icons needed for these snacks! Thanks go out to him!
- Borer Yummies now come in packs, containing 4 yummies in total. These yummies are randomized when added to packs and they vary in color, they could be green, red, yellow, or blue.
- Specific yummies get fun descriptions to go along with their color
- Toritose are cool

File: code/game/objects/items/weapons/storage/food.dm [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	..()
+	var/obj/item/weapon/reagent_containers/food/snacks/borer/B
+	for(var/i, i < 4, i++)
+		B = new(src)
+		B.icon_state = rand("redyum", "greenyum", "blueyum", "yellowyum")
+		switch(B.icon_state)
+			if("redyum")
+				B.desc = "A cherry flavoured yummie! This'll do the trick!"
+			if("blueyum")
+				B.desc = "A squishy berry-flavoured yummie! This'll leave your mouth full!"
```
Comment (ShadowDeath6) [inline code/game/objects/items/weapons/storage/food.dm:28]: flavor

File: code/game/objects/items/weapons/storage/food.dm [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+			if("greenyum")
+				B.desc = "A green yummie! It's ... moving."
+			if("yellowyum")
+				B.desc = "A banana flavoured yummie! Packed with lots of flavour!"
```
Comment (ShadowDeath6) [inline code/game/objects/items/weapons/storage/food.dm:32]: flavor

File: code/game/objects/items/weapons/storage/food.dm [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	for(var/i, i < 4, i++)
+		B = new(src)
+		B.icon_state = rand("redyum", "greenyum", "blueyum", "yellowyum")
+		switch(B.icon_state)
+			if("redyum")
+				B.desc = "A cherry flavoured yummie! This'll do the trick!"
+			if("blueyum")
+				B.desc = "A squishy berry-flavoured yummie! This'll leave your mouth full!"
+			if("greenyum")
+				B.desc = "A green yummie! It's ... moving."
```
Comment (ShadowDeath6) [inline code/game/objects/items/weapons/storage/food.dm:30]: is this lore?

File: code/game/objects/items/weapons/storage/food.dm (hunk 1 above)
Comment (Super3222) [reply code/game/objects/items/weapons/storage/food.dm:28]: Flavour is technically not incorrect, but it's just not favored by the english language.

File: code/game/objects/items/weapons/storage/food.dm (hunk 3 above)
Comment (Super3222) [reply code/game/objects/items/weapons/storage/food.dm:30]: No it's a joke. Green snack. Borer shaped. Would you have preferred: "A green yummie! it tastes green!".  Well that's boring tbh, and I like making things surprising.

**Comment by Dragonforcefive** [comment]
Lol. Advertisements say that Borer Yummies can give you cool borers...

**Comment by Super3222** [comment]
This is ready for review/merge, Cruix

**Comment by Notamaniac** [comment]
The red on those toritos are really really saturated. You might want to consider pulling back on the saturation and just use more brightness. Shading is a bit boring too, adding a touch of illumination to the packet to represent the crinkles and putting the light source to the top left mig-

I am literally describing how to make a crisp packet look more interesting

The fuck is wrong with my life

**Comment by AndySok** [comment]
Well , not your fault , Notamaniac.

Toritos sprite is just a recolored version of the normal chips.

**Comment by Time-Green** [comment]
Pictures?

**Pull request merged**